docker/vector.toml

```diff
@@ -0,0 +1,27 @@
+## Prometheus Host metrics
+
+[sources.host_metrics]
+type = "host_metrics"
+collectors = ["cpu", "disk", "load", "filesystem", "host", "memory", "network"]
+scrape_interval_secs = 15
+
+[sources.host_metrics.filesystem.mountpoints]
+includes = ["/"]
+
+[sources.host_metrics.disk.devices]
+includes = ["/dev/root", "nvme*", "tmpfs"]
+
+[sources.app]
+type = "prometheus_scrape"
+endpoints = [
+    "http://host.docker.internal:9181",
+    "http://host.docker.internal:9182",
+    "http://host.docker.internal:9183",
+]
+
```

Comment:
Oh! I didn't knew we could do this with vector! So we can use the same stack to push signer metrics (if they allow us to!)

Reply:
Yup, it's a great tool.